Master

shingoxy opened the pull request:
# 增加GC5603.c驱动并修改对应Kernel文件：
1. drivers/media/i2c/增加GC5603.c文件；
2. arch/arm64/boot/dts/rockchip/rk3588s-lubancat-csi2.dtsi修改Device tree；
3. arch/arm64/configs/lubancat_linux_rk3588_defconfig修改defconfig
4. drivers/media/i2c/Kconfig增加gc5603.c编译
5. drivers/media/i2c/Makefile增加gc5603.c编译
6. drivers/phy/rockchip/phy-rockchip-csi2-dphy.c修改后增加cphy支持

以上修改已经编译通过（2024年6月），将进行点亮验证。

shingoxy committed:
修改：     .version
	修改：     arch/arm64/boot/dts/rockchip/rk3588s-lubancat-csi2.dtsi
        修改：     arch/arm64/configs/lubancat_linux_rk3588_defconfig
	修改：     drivers/media/i2c/Kconfig
	修改：     drivers/media/i2c/Makefile
	新文件：   drivers/media/i2c/gc5603.c
	修改：     drivers/phy/rockchip/phy-rockchip-csi2-dphy.c
	修改：     logo.bmp
	修改：     logo_kernel.bmp

shingoxy committed:
Update README.md

shingoxy committed:
Merge branch 'stable-5.10-rk3588' into master

hejiawencc commented:
非常感谢 @shingoxy  提交 pull request，如果后续功能验证成功，请在这里说明。如果测试通过，我们将合并你提交的修改并将设备树的修改整理成设备树插件的形式。如果需要帮助，可以在这里交流或者联系我们的技术支持。

shingoxy committed:
修改：     .version
	修改：     arch/arm64/boot/dts/rockchip/rk3588s-lubancat-csi2.dtsi
	修改：     arch/arm64/configs/lubancat_linux_rk3588_defconfig
	修改：     drivers/media/i2c/gc5603.c

shingoxy committed:
Merge branch 'master' of https://github.com/shingoxy/kernel-RK3588s-5.10Stable

	.version

	修改：     README.md
	修改：     arch/arm64/boot/dts/rockchip/overlay/Makefile
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-5-cam0-imx415-1080p-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-5-cam1-imx415-1080p-overlay.dts
	修改：     arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-5-cam1-imx415-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-5-cam2-imx415-1080p-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-5-cam3-imx415-1080p-overlay.dts
	修改：     arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-5-cam3-imx415-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-5-cam4-imx415-1080p-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-5-cam5-imx415-1080p-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-i2c2-m0-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-pwm0-m0-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-pwm1-m0-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-spi1-m1-gpio-cs-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588-lubancat-uart5-m1-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588s-lubancat-4-cam0-imx415-1080p-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588s-lubancat-4-cam1-imx415-1080p-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588s-lubancat-4-cam2-imx415-1080p-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588s-lubancat-4-v1-xl9535-overlay.dts
	新文件：   arch/arm64/boot/dts/rockchip/overlay/rk3588s-lubancat-4-xl9535-overlay.dts
	修改：     arch/arm64/boot/dts/rockchip/rk3588-lubancat-5-cam1-imx415.dtsi
	修改：     arch/arm64/boot/dts/rockchip/rk3588-lubancat-5-cam3-imx415.dtsi
	修改：     arch/arm64/boot/dts/rockchip/rk3588-lubancat-5-csi.dtsi
	修改：     arch/arm64/boot/dts/rockchip/rk3588-lubancat-5.dts
	修改：     arch/arm64/boot/dts/rockchip/rk3588-lubancat-5io.dts
	修改：     arch/arm64/boot/dts/rockchip/uEnv/uEnvLubanCat4-V1.txt
	修改：     arch/arm64/boot/dts/rockchip/uEnv/uEnvLubanCat4.txt
	修改：     arch/arm64/boot/dts/rockchip/uEnv/uEnvLubanCat5.txt
	修改：     arch/arm64/configs/lubancat_linux_rk3588_defconfig
	修改：     drivers/Kconfig
	修改：     drivers/Makefile
	新文件：   drivers/ebf_module/Kconfig
	新文件：   drivers/ebf_module/Makefile
	新文件：   drivers/ebf_module/dht11/Makefile
	新文件：   drivers/ebf_module/dht11/dht11.c
	新文件：   drivers/ebf_module/dht11/dht11.h
	新文件：   drivers/ebf_module/dht11/test_app.c
	新文件：   drivers/ebf_module/ds18b20/Makefile
	新文件：   drivers/ebf_module/ds18b20/ds18b20.c
	新文件：   drivers/ebf_module/ds18b20/ds18b20.h
	新文件：   drivers/ebf_module/ds18b20/ds18b20.sh
	新文件：   drivers/ebf_module/ds18b20/test_app.c
	修改：     drivers/media/i2c/imx415.c
	修改：     drivers/net/phy/Kconfig
	修改：     drivers/net/phy/Makefile
	修改：     drivers/net/phy/jlsemi-core.h
	新文件：   drivers/net/phy/linkyum.c
	修改：     drivers/rknpu/Makefile
	新文件：   drivers/rknpu/include/rknpu_devfreq.h
	修改：     drivers/rknpu/include/rknpu_drv.h
	修改：     drivers/rknpu/include/rknpu_gem.h
	修改：     drivers/rknpu/include/rknpu_ioctl.h
	修改：     drivers/rknpu/include/rknpu_iommu.h
	修改：     drivers/rknpu/include/rknpu_job.h
	修改：     drivers/rknpu/include/rknpu_mem.h
	修改：     drivers/rknpu/rknpu_debugger.c
	新文件：   drivers/rknpu/rknpu_devfreq.c
	修改：     drivers/rknpu/rknpu_drv.c
	修改：     drivers/rknpu/rknpu_gem.c
修改：     drivers/rknpu/rknpu_iommu.c
	修改：     drivers/rknpu/rknpu_job.c
	修改：     drivers/rknpu/rknpu_mem.c
	修改：     drivers/rknpu/rknpu_reset.c
	修改：     drivers/soc/rockchip/rockchip_opp_select.c
	新文件：   include/dt-bindings/phy/jlsemi-dt-phy.h
	修改：     include/linux/version_compat_defs.h
	修改：     include/soc/rockchip/rockchip_opp_select.h

shingoxy committed:
修改：     README.md

shingoxy committed:
Update README.md

shingoxy commented:
<img width="640" alt="image" src="https://github.com/LubanCat/kernel/assets/36981700/a0239be1-2aa7-437b-a3e7-f58a8764c66b">
0628成功点亮

shingoxy commented:
虽然已经点亮，但是缺少3A文件，3A似乎不能正常工作@hejiawencc 是否有对应的3A SDK包可以使用？

shingoxy committed:
修改：     arch/arm64/boot/dts/rockchip/rk3588s-lubancat-csi2.dtsi
	修改：     arch/arm64/configs/lubancat_linux_rk3588_defconfig
	修改：     arch/arm64/configs/rockchip_defconfig
	修改：     arch/arm64/configs/rockchip_linux_defconfig
	修改：     drivers/media/i2c/gc5603.c

shingoxy committed:
Merge branch 'master' of https://github.com/shingoxy/kernel-RK3588s-5.10Stable

shingoxy committed:
修改：     arch/arm64/configs/lubancat_linux_rk3588_defconfig
	修改：     arch/arm64/configs/rockchip_defconfig
	修改：     arch/arm64/configs/rockchip_linux_defconfig
	修改：     drivers/media/i2c/gc5603.c

shingoxy commented:
@hejiawencc 修改dts，3A已经正常工作，但是iqfile是copy的os04a10的，颜色不正常。

shingoxy commented:
![image](https://github.com/LubanCat/kernel/assets/36981700/ef07a892-2266-40d0-bd7d-5f90066c0157)

shingoxy committed:
Update README.md

shingoxy committed:
修改：     arch/arm64/boot/dts/rockchip/rk3588s-lubancat-4.dts
	修改：     arch/arm64/boot/dts/rockchip/rk3588s-lubancat-40pin.dtsi
	修改：     arch/arm64/boot/dts/rockchip/rk3588s-lubancat-csi2.dtsi
	修改：     arch/arm64/configs/lubancat_linux_rk3588_defconfig
	修改：     arch/arm64/configs/rockchip_defconfig
	修改：     arch/arm64/configs/rockchip_linux_defconfig
	修改：     drivers/media/i2c/Kconfig
	修改：     drivers/media/i2c/Makefile
	修改：     drivers/media/i2c/gc5603.c

shingoxy committed:
新文件：   drivers/media/i2c/gc8613.c

shingoxy committed:
Merge branch 'master' of https://github.com/shingoxy/kernel-RK3588s-5.10Stable

shingoxy committed:
Update README.md

shingoxy committed:
修改：     arch/arm64/boot/dts/rockchip/rk3588s-lubancat-csi2.dtsi
	修改：     arch/arm64/configs/lubancat_linux_rk3588_defconfig
	修改：     arch/arm64/configs/rockchip_defconfig
	修改：     arch/arm64/configs/rockchip_linux_defconfig
	修改：     drivers/media/i2c/Kconfig
	修改：     drivers/media/i2c/Makefile
	修改：     drivers/media/i2c/gc5603.c
	新文件：   drivers/media/i2c/ov50h40.c

shingoxy committed:
Update README.md
